estate/__manifest__.py

```diff
@@ -0,0 +1,10 @@
+{
+    'name': 'Real Estate',
+    'depends': ['base'],
+    'data': ['security/ir.model.access.csv',
+            'views/estate_property_views.xml',
+            'views/estate_menus.xml'],
```

Comment:
let's be coherent with the rest of odoo modules 👀 
```suggestion
    'data': [
        'security/ir.model.access.csv',
        'views/estate_property_views.xml',
        'views/estate_menus.xml'
    ],
```